test/fixtures/flow/type-annotations/136/actual.js

```diff
@@ -0,0 +1,4 @@
+type A = {
+	p: {},
+	...{},
+};
```

Comment:
Right, we "reserved" the syntax in 0.40, but 0.42 extends support to all type syntax and adds support.